src/pages/health-declaration/cat-home-test/health-declaration.cat-home-test.page.ts

```diff
@@ -242,19 +241,7 @@ export class HealthDeclarationCatHomeTestPage extends BasePageComponent {
           this.store$.dispatch(new ProvisionalLicenseNotReceived());
         }
         this.store$.dispatch(new ContinueFromDeclaration());
-        this.navController.push(CAT_HOME_TEST.BACK_TO_OFFICE_PAGE).then(() => {
-          this.navController.getViews().forEach((view) => {
-            if (includes([
-              CAT_HOME_TEST.TEST_REPORT_PAGE,
-              CAT_HOME_TEST.DEBRIEF_PAGE,
-              CAT_HOME_TEST.PASS_FINALISATION_PAGE,
-              CAT_HOME_TEST.HEALTH_DECLARATION_PAGE,
-            ],
-              view.id)) {
-              this.navController.removeView(view);
-            }
-          });
-        });
+        this.navController.push(CONFIRM_TEST_DETAILS);
```

Comment:
Minor thing, but if we are using navController.push here can we use async/await as push returns a promise - same goes for other usages